src/App.js

```diff
@@ -20,14 +20,51 @@ class App extends Component{
   constructor(props){
     super(props)
     this.state = {
-      cats: mockCats
+      cats: []
     }
   }
 
+  componentDidMount(){
+    this.readCat()
+  }
+
   createCat = (cat) => {
     console.log(cat)
   }
 
+  readCat = () => {
```

Comment:
Make sure you remove this createCat ^^ method. This was only the placeholder until you added the fetch functionality which looks to be on line 42.